src/script.js

```diff
@@ -42,71 +46,87 @@ var gianniAccordion = (function(){
       el.content.addEventListener(this.transitionendevent, bindEvent);
 
       el.isOpen = true;
+      el.wrapper.classList.remove(this.collapsedClass);
       el.content.style.height = el.content.scrollHeight + "px";
     }
 
-    collapse(el){
+    collapse(el) {
 
-      function endTransition(ev){     
-        if(ev.target != el.content) return
+      function endTransition(ev) {
+        if (ev.target != el.content) return
         el.content.removeEventListener(this.transitionendevent, bindEvent);
 
-        if(el.isOpen) return
+        if (el.isOpen) return
 
         this.fire("elementClosed", el);
+        el.trigger.setAttribute('aria-expanded', 'false');
+        el.content.setAttribute('aria-hidden', 'true');
```

Comment:
I due aria vengono ripetuti uguali in un'altra parte, così come i loro opposti (true e false). Possiamo centralizzare con due metodi setCollapsedAria() e setExpandedAria()